contracts/math/clamm.ral

```diff
@@ -337,10 +350,114 @@ Contract CLAMM(uints: Uints) extends Log(uints){
     }
 
     pub fn toFee(feeGrowth: U256, liquidity: U256) -> U256 {
-        return uints.toU256(uints.bigMulDiv256(feeGrowth, liquidity, one(FeeGrowthScale + LiquidityScale)))
+        return toU256(bigMulDiv256(feeGrowth, liquidity, one(FeeGrowthScale + LiquidityScale)))
     }
 
     pub fn feeGrowthFromFee(liquidity: U256, fee: U256) -> U256 {
-        return uints.toU256(uints.bigMulDiv256(fee, one(LiquidityScale + FeeGrowthScale), liquidity))
+        return toU256(bigMulDiv256(fee, one(LiquidityScale + FeeGrowthScale), liquidity))
+    }
+
+    pub fn getLiquidityByX(x: U256, lowerTick: I256, upperTick: I256, currentSqrtPrice: U256, roundingUp: Bool) -> SingleTokenLiquidity {
```

Comment:
Is there an option to define all off-chain functions in a separate contract (never to be deployed)? This way, we can avoid increasing the size of the deployed contract, thereby optimizing the cost of deployment and similar factors